api/call.go

```diff
@@ -126,26 +127,24 @@ func (ar *Runner) callHandler() func(http.ResponseWriter, *http.Request) {
 		ctx, span := instracer.StartSpan(ctx, "callHandler")
```

Comment:
function "callHandler" take 100 lines. Divide it into subfunctions

Reply:
it is less then 100 lines